src/api/app/views/webui/request/changes.html.haml

```diff
@@ -15,6 +15,13 @@
                   actions_count: @supported_actions.count, active_tab: @active_tab }
     .container.p-4
       .tab-content.sourcediff
+        - if @diff_to_superseded
+          You're not reviewing the full diff of
```

Comment:
> How about `You are viewing changes to [superseded request #xyz]. This only represents a small part of changes which are included in this request. [See full changes for request #abc]`

I like it!

Reply:
> It might also be a good idea to always display the information about being able to see the comparison to superseded here

You mean, don't hide the link about the diff to the superseded?

Reply:
Alright

Reply:
I made changes, please review again please